casper-cep78-js-client 1.3.0 (#207)
casper-ecosystem/enhanced-nft: client-js/README.md

4. If you want to start working with a previously installed contract, use the `setContractHash(contractHash)` method.

> NOTE: Since version `1.3` both `casper-js-sdk` and `@make-software/ces-js-parser` are peerDependencies, so if you're using npm version `<7` you might need to install both deps manually.

Reviewer: ```suggestion
NOTE: Since version `1.3` both `casper-js-sdk` and `@make-software/ces-js-parser` are peer dependencies. If you are using npm version `<7` you may need to install both dependencies manually.
```
